lib/rules.ts

```diff
@@ -18,6 +20,8 @@ export type RunContext = {
   urlPattern?: string,
 }
 
+export type ElementSelector = Selector
```

Comment:
Since we now support `any`, do we still need to support Selector as string[], and all the other changes in this file and rule-executors.ts?

Reply:
We still need `string[]`, as puppeteer reply may emit a chained selector, where we run a series of selectors on subsets of the dom.